src/assets.ts

```diff
@@ -258,24 +258,17 @@ export class AssetGenerator {
     }
 
     public createLaunchJson(isWebProject: boolean): any {
```

Comment:
We might want to have this return a JSON'ified string instead of an array of objects. My reasons --
1. minor, but 'any' is a shitty type
2. Hopefully someday we will do a similar fix for adding comments inside the launch target. Bonus points if you want to fix this - I am specifically thinking of 'terminal', since that has gotten more than a few questions, and 'program', since that is probably the most important thing, and it may not be obvious that that needs to be updated after retargeting your app at a new version of the framework.